pyproject.toml

```diff
@@ -65,4 +50,235 @@ known_twisted = ["twisted", "OpenSSL"]
 multi_line_output = 3
 include_trailing_comma = true
 combine_as_imports = true
+skip_gitignore = true
 
+[tool.poetry]
+name = "matrix-synapse"
+version = "1.54.0rc1"
```

Comment:
This is now out-of-date.

Reply:
I guess we should aim this at the 1.57 release? Would be nice to mark this as a prerelease somehow if possible. Can't remember the right notation for this though, will have to check https://peps.python.org/pep-0440/

Reply:
Merged in develop, resolving this. Now that develop has a `pyproject.toml` file with this metadata, we shouldn't need to worry about this further in this PS.